### PR TITLE
adding iam-root-key-access rule

### PR DESCRIPTION
**Pull request by coolguru**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -99,6 +99,75 @@
...
+      user_name
+      access_key_1_active
+      access_key_1_last_used_date
+      access_key_1_last_used_service
+      access_key_1_last_used_region
+      access_key_2_active
+      access_key_2_last_used_date
+      access_key_2_last_used_service
+      access_key_2_last_used_region
+      relates_to {
```
Comment (armaneous) [inline services/config.rb]: This is potentially big, I think. Might be better to scope to a handful of types.

File: services/config.rb [hunk 2]
```diff
@@ -99,6 +99,75 @@
...
+        <%= default_predicates %>
+      }
+    }
+  }
+  QUERY
+  meta_rule_node_triggers({
+                              'user' => ['user_name', 'access_key_1_active', 'access_key_1_last_used_date', 'access_key_2_active', 'access_key_2_last_used_date']
```
Comment (armaneous) [inline services/config.rb]: I don't think you need the triggers for `*_last_used_date` since it's not used for the violation query above.

Comment (scvenema) [reply services/config.rb]: Lets just fix the trigger in the new repo. I'll start a JIRA task on this.

File: services/config.rb (hunk 1 above)
Comment (scvenema) [reply services/config.rb]: Yes, we should keep all those predicates for the visualize, but remove them from the violation query.

**Pull request merged**